README.txt

```diff
@@ -235,5 +235,8 @@ FUTURE WORK
 - Allow SoftwareSerial or HardwareSerial to be used interchangeably?
 - CRC_XMODEM needs debugging, trying to test using a basic sketch and lrzsz
   I need a way to spy on what lrzsz is sending/receiving
-- Raspberry Pi Pico (RP2040) port
 - A standardised performance testing sketch
+
+Credits
```

Comment:
I don't know much about best practices regarding giving credit to contributors, especially within the MIT and arduino ecosystems.
If you could point me to any resources/examples that I can use to read up on that would be greatly appreciated